### ISSUE TITLE
训练问题

### ISSUE THREAD

**Issue by l976308589** [Guillem96/efficientdet-tf#2]
尊敬的作者，您好。
在训练的时候，一直显示如下问题

 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = -1.000
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = -1.000
creating index...
index created!
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=0.00s).
请问是什么原因

**Comment by Guillem96** [comment]
For how many epochs did you run the training? Can you provide more details?

**Comment by l976308589** [comment]
Thanks for your prompt reply.
format is VOC
learning_rate is 0.001
epochs is 200
Epoch[0] loss: 2.854052 clf. loss: 1.148139 reg. loss: 1.705913 
Epoch[0] loss: 2.895702 clf. loss: 1.149788 reg. loss: 1.745914 
Epoch[0] loss: 2.896174 clf. loss: 1.145899 reg. loss: 1.750275 
Epoch[0] loss: 2.875462 clf. loss: 1.137185 reg. loss: 1.738277 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Corrupt JPEG data: 2 extraneous bytes before marker 0xd9
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 
Epoch[0] loss: nan clf. loss: inf reg. loss: nan 


maybe something error in my pictures

**Comment by Guillem96** [comment]
Ir seems so. Maby when you load a corrupt image.you are crating tensors with NAN values.
Try with other images.

**Comment by karolmajek** [comment]
check annotations (below zero, or equal/larger than w/h)
